src/runtime/com.rs

```diff
@@ -0,0 +1,44 @@
+use crate::*;
+
+/// Initializes COM for use by the calling thread for the multi-threaded apartment (MTA).
+pub fn initialize_mta() -> Result<()> {
+    unsafe {
+        CoInitializeEx(0, 0).ok() // COINIT_APARTMENTTHREADED
+    }
+}
+
+/// Initializes COM for use by the calling thread for a single-threaded apartment (STA).
+pub fn initialize_sta() -> Result<()> {
+    unsafe {
+        CoInitializeEx(0, 2).ok() // COINIT_MULTITHREADED
```

Comment:
Nit: can you use a constant instead of the raw number plus the comment in all these cases?